kubernetes-cs-guestbook/components/README.md

```diff
@@ -0,0 +1,66 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
```

Comment:
Nit: We try to use a slightly more structured template for READMEs, like [here](https://github.com/pulumi/examples/blob/master/azure-ts-aks-keda/README.md)

Reply:
Good point - this was copied from the corresponding TS template - so sounds like we should update both (or open an issue to create consistency here for all existing templates).

Reply:
Opened https://github.com/pulumi/examples/issues/493 to just to this consistently in a single pass in the near future.